JYH/5Day/제일 작은 수 제거하기.js

```diff
@@ -0,0 +1,11 @@
+function solution(arr) {
+    if (arr.length === 1) return [-1];
+    
+    let answer = arr.reduce((acc, cur) => acc < cur ? acc : cur);
```

Comment:
```suggestion
    const answer = arr.reduce((acc, cur) => acc < cur ? acc : cur);
```
@jungyoonho128 https://github.com/modern-agile-team/9term-assignment/pull/38#discussion_r1876351432 필독해주세요 :)

Reply:
넵 알겠습니다!